src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,88 @@
+package duke;
+
+/**
+ * This class handles the display for user interaction.
+ */
+public class Ui {
+
+    protected final String ADDED = "oke i added this:";
+    protected final String DELETED = "oke i deleted this:";
+    protected final String MARKED = "oke this is done now:";
+    protected final String UNMARKED = "oke this is undone now:";
+
+
+    /**
+     * Constructor for the Ui class.
+     */
+    public Ui () {
+
+    }
+
+
+    /**
+     * Prints an exit message.
+     *
+     * @return String message.
+     */
+    public String showExit() {
+        return "bye see u\n";
+    }
+
+    /**
+     * Displays a task.
+     *
+     * @param message Message to be printed.
+     * @param task Task to be displayed.
+     * @return String displaying the task.
+     */
+    public String displayTask(String message, Task task) {
+        return message + "\n" + task + "\n";
+    }
+
+    /**
+     * Prints the entire list of tasks.
+     *
+     * @param taskList Tasklist to be printed.
+     * @return String containing all the tasks.
+     */
+    public String printTasks(TaskList taskList) {
+        return "here! ur tasks:\n" + taskList + "\n";
+    }
+
+    /**
+     * Displays the total amount of tasks
+     * in the taskList.
```

Comment:
```suggestion
     * Displays the total amount of tasks in the taskList.
     * 
```
I think you can consider shifting the second line up.